api/capi/capi-nnstreamer.pc.in

```diff
@@ -9,6 +9,6 @@ includedir=@INCLUDE_INSTALL_DIR@
 Name: tizen-api-nnstreamer
 Description: NNStreamer API for Tizen
 Version: @VERSION@
-Requires:
-Libs: -L${libdir}
-Cflags: -I${includedir}
+Requires: nnstreamer
```

Comment:
No. this shouldn't require nnstreamer.pc

Reply:
Right!